### ISSUE TITLE
calendar 

### ISSUE THREAD

**Issue by NOsteyr** [BobTheShoplifter/HomeAssistant-Posten#26]
I would like to change the name 'levering' that appears in the calendar to something else. Perhaps it could be an option when reconfiguring or adding the addon?

**Comment by BobTheShoplifter** [comment]
What would you liked it changed to? Not sure how options work in hass

**Comment by gf-a** [comment]
I think it should be named "postlevering"

**Comment by Olen** [comment]
I agree.  I will create a PR shortly, to add an option for setting the name.
